packages/components/autocomplete/src/Autocomplete.tsx

```diff
@@ -99,14 +95,6 @@ function _Autocomplete<ItemType>(
     testId = 'cf-autocomplete',
   } = props;
 
-  const formProps = useFormControl({
```

Comment:
Don't we want to integrate form control with autocomplete?

Reply:
It has support out of the box, since we are using `TextInput` that's watching context value from `FormControl`.

Reply:
ah, that's nice!